api/src/main/java/org/pmiops/workbench/actionaudit/auditors/AdminAuditorImpl.kt

```diff
@@ -0,0 +1,55 @@
+package org.pmiops.workbench.actionaudit.auditors
+
+import org.pmiops.workbench.actionaudit.ActionAuditEvent
+import org.pmiops.workbench.actionaudit.ActionAuditService
+import org.pmiops.workbench.actionaudit.ActionType
+import org.pmiops.workbench.actionaudit.AgentType
+import org.pmiops.workbench.actionaudit.TargetType
+import org.pmiops.workbench.db.model.DbUser
+import org.pmiops.workbench.model.AccessReason
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import java.time.Clock
+import java.util.logging.Logger
+import javax.inject.Provider
+
+@Service
+class AdminAuditorImpl @Autowired
+constructor(
+    private val userProvider: Provider<DbUser>,
+    private val actionAuditService: ActionAuditService,
+    private val clock: Clock,
+    @Qualifier("ACTION_ID") private val actionIdProvider: Provider<String>
+) : AdminAuditor {
+
+    override fun fireViewNotebookAction(workspaceNamespace: String, workspaceName: String, accessReason: AccessReason, notebookFilename: String) {
+        val dbUser = userProvider.get()
+        val actionId = actionIdProvider.get()
+
+        val props = mapOf(
+            "Workspace Namespace" to workspaceNamespace,
```

Comment:
So far the convention has been `snake_case` for [property names](https://github.com/all-of-us/workbench/blob/7d64640cf5bfcfb5755438e6e89ff88a8b24ba45/api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/WorkspaceTargetProperty.kt#L12), though to be fair I cut them up in the client for display. I have a slight preference for consistency on the DB side.

Reply:
fixed